dts/bindings/rtc/nxp,irtc.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: IRTC
+
+compatible: "nxp,irtc"
+
+include:
+  - name: rtc.yaml
+  - name: rtc-device.yaml
+
+properties:
+  reg:
+    required: true
+
+  clock-src:
+    type: int
+    required: true
```

Comment:
you can add enum here to ensure only valid values are entered:
```

  clock-src:
    type: int
    required: true
    enum:
      - 0
      - 1
    description: |
      The input clock select for IRTC.
      0 <- 16.384 kHz
      1 <- 32.768 kHz
```

Reply:
I wasn't aware, I will add this to the binding in the next update.